docs/src/main/tut/docs/README.md

```diff
@@ -80,9 +83,11 @@ Abstract definitions it's all it takes to start building programs that support s
 The example below combines both algebras to produce a more complex program
 
 ```tut:book
+import cats.syntax.cartesian._
```

Comment:
The cats documentation choose to always use `import cats.implicits._` instead of the piecemeal imports as it is more beginner friendly, should we do the same?